server/main.js

```diff
@@ -1,41 +1,140 @@
+      }),
+      ClimatesCollection.insertAsync({ 
+        name: 'Méditerranéen', 
+        description: 'Climat chaud et sec en été, doux en hiver',
+        temperatureRange: { min: 10, max: 35 }
+      })
+    ]);
+  }
 
-    await insertLink({
-      title: 'Discussions',
-      url: 'https://forums.meteor.com',
-    })
+  if ((await PlantsCollection.find().countAsync()) === 0) {
+    await Promise.all([
+      PlantsCollection.insertAsync({ 
+        name: 'Tomate',
+        maintenances: [
+          {
+            _id: Random.id(),
+            name: "Arrosage",
+            frequency: 5,
+          },
+          {
+            _id: Random.id(),
+            name: "Sulfatage",
+            frequency: 150,
+          }
+        ]
+      }),
+      PlantsCollection.insertAsync({ 
+        name: 'Basilic',
+        maintenances: [
+          {
+            _id: Random.id(),
+            name: "Arrosage",
+            frequency: 5,
+          },
+          {
+            _id: Random.id(),
+            name: "Anti-limaces",
+            frequency: 30,
+          }
+        ]
+      })
+    ]);
+  }
 
-    await insertLink({
-      title: 'Test link',
-      url: 'https://smash-map.com',
-    })
+  if ((await UsersCollection.find().countAsync()) === 0) {
+    await Promise.all([
+      UsersCollection.insertAsync({ 
+        name: 'Marco Paladini',
+        email: 'alice.dupont@email.com',
+        gardens: [
+          {
+            _id: Random.id(),
+            name: 'Jardin Principal',
+            climateId: 'id',
+            plants: [
+              {
+                _id: Random.id(),
+                plantId: 'id',
+                position: { x: 2, y: 2 },
+                sensors: [
+                  {
+                    _id: Random.id(),
+                    sensorId: 'id',
+                    datasUrl: "url"
+                  }
+                ],
+                maintenancesDone: [
+                  {
+                    _id: Random.id(),
+                    maintenanceId: "id",
+                    date: new Date()
+                  }
+                ]
+              }
+            ]
+          }
+        ]
+      }),
+      UsersCollection.insertAsync({ 
+        name: 'Marc Bolan',
+        email: 'bob.martin@email.com',
+        gardens: [
+          {
+            _id: Random.id(),
+            name: 'Jardin Urbain',
+            climateId: 'id',
+            plants: [
+              {
+                _id: Random.id(),
+                plantId: 'id',
+                position: { x: 2, y: 2 },
+                sensors: [
+                  {
+                    _id: Random.id(),
+                    sensorId: 'id',
+                    datasUrl: "url"
+                  }
+                ],
+                maintenancesDone: [
+                  {
+                    _id: Random.id(),
+                    maintenanceId: "id",
+                    date: new Date()
+                  }
+                ]
```

Comment:
This should refer to an actual maintenance_id inside the maintenances collection